### PR TITLE
Adds the other half of the fox infusion because Arm is a coward

### PR DESCRIPTION
**Pull request by MrMelbert**
## About The Pull Request

Adds the other half of the fox infusion

## Why It's Good For The Game

because Arm is a coward

## Changelog

:cl: Melbert
add: Fox infusion has more content related to it now
/:cl:

### CODE REVIEW

File: code/datums/sprite_accessories.dm [hunk 1]
```diff
@@ -1781,6 +1781,13 @@
 	icon_state = "default"
 	color_src = HAIR_COLOR
 
+/datum/sprite_accessory/tails/human/fox
```
Comment (DATA-xPUNGED) [inline code/datums/sprite_accessories.dm:1784]: hell yeah

**Comment by carlarctg** [comment]
Add one tail per fox infusef until you reach 9 and become a powerful waifu

**Comment by norsvenska** [comment]
the wagging sprite looks kinda stiff

**Comment by CCC23** [comment]
finally I can become a true degenerate

**Comment by DrAmazing343** [comment]
WE'RE ACTUALLY SO FUCKING BACK IT IS GENUINELY UNREAL LET'S GOOOOOOOOOOOO

File: code/modules/surgery/organs/external/tails.dm [hunk 2]
```diff
@@ -169,6 +169,19 @@
...
+	bodypart_overlay = /datum/bodypart_overlay/mutant/tail/cat/fox
+	dna_block = null
+	sprite_accessory_override = /datum/sprite_accessory/tails/human/fox
+
+///Fox tail bodypart overlay
+/datum/bodypart_overlay/mutant/tail/cat/fox
+
+/datum/bodypart_overlay/mutant/tail/cat/fox/New()
+	. = ..()
+	set_appearance(/datum/sprite_accessory/tails/human/fox)
```
Comment (Time-Green) [inline code/modules/surgery/organs/external/tails.dm:183]: Why this? This should be handled by organ insert

Comment (MrMelbert) [reply code/modules/surgery/organs/external/tails.dm:183]: I forgot to delete it

Comment (MrMelbert) [inline code/modules/surgery/organs/external/tails.dm:183]:
```suggestion
```

**Pull request merged**